{"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"writeError","kind":"identifier"}]}]}],"identifier":{"url":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError\/writeError","interfaceLanguage":"swift"},"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"writeError","kind":"identifier"}],"symbolKind":"case","externalID":"s:10ScanbotSDK24SBSDKImageProcessorErrorO05writeE0yA2CmF","role":"symbol","title":"SBSDKImageProcessorError.writeError","modules":[{"name":"ScanbotSDK"}],"roleHeading":"Case"},"abstract":[{"type":"text","text":"An unknown error occured while writing an image to a file."}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK","doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError"]]},"variants":[{"paths":["\/documentation\/scanbotsdk\/sbsdkimageprocessorerror\/writeerror"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK/SBSDKImageProcessorError":{"navigatorTitle":[{"text":"SBSDKImageProcessorError","kind":"identifier"}],"url":"\/documentation\/scanbotsdk\/sbsdkimageprocessorerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"SBSDKImageProcessorError","kind":"identifier"}],"type":"topic","role":"symbol","kind":"symbol","title":"SBSDKImageProcessorError","abstract":[{"text":"An enum of possible errors when processing images.","type":"text"}],"identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError"},"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK/SBSDKImageProcessorError/writeError":{"identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError\/writeError","type":"topic","title":"SBSDKImageProcessorError.writeError","abstract":[{"text":"An unknown error occured while writing an image to a file.","type":"text"}],"role":"symbol","url":"\/documentation\/scanbotsdk\/sbsdkimageprocessorerror\/writeerror","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"writeError","kind":"identifier"}]},"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK":{"url":"\/documentation\/scanbotsdk","abstract":[{"text":"Version: 5.2.2","type":"text"}],"identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK","kind":"symbol","role":"collection","type":"topic","title":"ScanbotSDK"}}}